src/components/__tests__/page-details.test.jsx

```diff
@@ -64,7 +64,7 @@ describe('page-details', () => {
 
     await waitFor(() => expect(mockApi.getPage).toHaveBeenCalled());
     expect(mockApi.sampleVersions).toHaveBeenCalled();
-    expect(document.title).toBe('Scanner | http://www.ncei.noaa.gov/news/earth-science-conference-convenes');
+    await waitFor(() => expect(document.title).toBe(`Scanner | ${simplePage.url}`));
```

Comment:
I don’t think this is actually related to the router updates, but I noticed this was flaky while working on the v6 upgrade, and hit it a few more times during this one, so I figured I’d just make this a little less race-y.